app/schemas/subscriptions/play.coffee

```diff
@@ -124,6 +124,7 @@ module.exports =
   'level:show-victory': c.object {required: ['showModal']},
     showModal: {type: 'boolean'}
     manual: { type: 'boolean' }
+    capstoneVictory: { type: 'boolean' }
```

Comment:
This is nice! Is this enabling us to gain more context about which show-victory modal to show?

Maybe this isn't the right place to discuss the architecture, but I'm interested in the benefit of passing this state in the event?

Reply:
This is how we say "this level was just completed, but only progress to the next capstone stage, because we are not fully done with the level yet". 

The way the code is structured for goal completion and through a victory screen, we need to pass along this information so the next function knows whether to go to victory like usual, or to go to the next cinematic - for example.